core/drivers/crypto/caam/acipher/caam_rsa.c

```diff
@@ -85,7 +85,7 @@ static uint8_t caam_era;
  *
  * @key  RSA keypair
  */
-static void do_keypair_free(struct rsa_keypair *key)
+static void do_free_keypair(struct rsa_keypair *key)
```

Comment:
And so you have to change `.free_keypair = &do_keypair_free,` to `.free_keypair = &do_free_keypair,`

Reply:
minor: looks not very nice these 2 functions almost have the same names: `do_keypair_free()` and `do_fre_keypair`.

Reply:
There are not doing the same and this is internal to the caam driver. This doesn't cause a problem for me